app/src/main/java/com/learning/catz/facts/FactsViewModel.kt

```diff
@@ -18,8 +19,13 @@ class FactsViewModel : ViewModel(), LifecycleObserver {
     private fun init() {
         _loading.value = true
         viewModelScope.launch {
-            _loading.value = false
-            storage.facts.collect { _facts.value = it }
+            requestCuriousities()
         }
     }
+
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
```

Comment:
Vc precisa realmente de um visible for testing aqui?
Creio que dá pra testar o comportamento dele por reflexo da chamada do init, concorda?